### DISCUSSION TITLE
Setup Airflow authentication using Azure AD

### DISCUSSION THREAD

**Discussion by ebebpl** [apache/airflow#30256] [Q&A] [answered]
Hello,

We have installed Airflow 2.5.1 using the official Helm chart. We would like to configure the authentication to the Airflow UI using Azure AD. What is the best way to configure the Azure AD authentication and is there any documentation to configure this.
Thanks for any help!

**Comment by potiuk** [comment, marked as answer]
Your best shot is https://flask-appbuilder.readthedocs.io/en/latest/ - we are using FAB for authentication and all the cofiguration options and details of all integration options come from there.
